product_supplierinfo_for_customer/views/product_view.xml

```diff
@@ -120,11 +97,8 @@
                 <group attrs="{'invisible': [('product_variant_count','&lt;=',1)]}">
```

Comment:
I think you are able to remove most of these overwrites with the refactoring. Take also into account the Odoo paradigm of not splitting variants supplierinfo with template ones.